biztalk/technical-guides/checklist-optimizing-performance-on-hyper-v.md

```diff
@@ -23,15 +23,15 @@ The following considerations apply when running BizTalk Server and/or the [!INCL
 
 ## Optimize Hyper-V Performance
  Use the following general guidelines to configure Hyper-V for optimal performance.
 
 |Step|Reference|
 |----------|---------------|
-|Apply recommended guidance for performance tuning virtualization servers. **Note:**  For the test scenarios described in [Testing BizTalk Server Virtualization Performance](~/technical-guides/testing-biztalk-server-virtualization-performance.md), the configuration options that were applied are described in the “Physical Infrastructure Specifics” and the “Virtualization Specifics” sections of the [Test Scenario Overview](~/technical-guides/test-scenario-overview.md) topic.|“Performance Tuning for Virtualization Servers” section of the “Performance Tuning Guidelines for Windows Server 2008 R2” document available at [https://go.microsoft.com/fwlink/?LinkID=202087](/windows-server/administration/performance-tuning/).|
+|Apply recommended guidance for performance tuning virtualization servers. **Note:**  For the test scenarios described in [Testing BizTalk Server Virtualization Performance](~/technical-guides/testing-biztalk-server-virtualization-performance.md), the configuration options that were applied are described in the “Physical Infrastructure Specifics” and the “Virtualization Specifics” sections of the [Test Scenario Overview](~/technical-guides/test-scenario-overview.md) topic.|“Performance Tuning for Virtualization Servers” section of the “Performance Tuning Guidelines for Windows Server 2008 R2” document available [here](/windows-server/administration/performance-tuning/).|
 |Close any Virtual Machine Connection windows that aren’t being used.|The Virtual Machine Connection window(s) displayed when double-clicking a virtual machine name in the Hyper-V manager consume resources that could be otherwise utilized.|
```

Comment:
```suggestion
| Close any Virtual Machine Connection windows that aren't in use. | In the Hyper-V manager, when you double-select a virtual machine name, Virtual Machine Connection windows appear. These windows consume resources that could otherwise be free for use. |
```